web_development_101/git_basics/introduction_to_git.md

```diff
@@ -29,7 +29,7 @@ By the end of this lesson, you should be able to do the following:
 
 <div class="lesson-content__panel" markdown="1">
 
-  1. Read Chapter 1.1 through 1.4 in [this book about version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control) to learn the differences between local, centralized, and distributed version control systems.
+  1. Read Chapter 1.1 through to 1.4 in [this book about version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control) to learn the differences between local, centralized, and distributed version control systems.
```

Comment:
I don't think this is necessary. I think the added "to" implies that you should stop at 1.4 instead of carrying forward and stopping at 1.5.